content/docs/command-reference/plots/index.md

```diff
@@ -216,7 +216,7 @@ Difference in this metric between the current project version and the previous
 commit:
 
 ```dvc
-$ dvc plots diff -d logs.csv HEAD^
+$ dvc plots diff --targets logs.csv -- HEAD^
```

Comment:
Thanks! Easier to specify the rev first and `--targets` later I think. I'll update it separately.